data/api/client-server/relations.yaml

```diff
@@ -0,0 +1,170 @@
+      parameters:
+        - in: path
+          type: string
+          name: roomId
+          description: The ID of the room the supposed parent event is in.
+          required: true
+          x-example: "!636q39766251:matrix.org"
+        - in: path
+          type: string
+          name: eventId
+          description: The supposed parent event ID.
+          required: true
+          x-example: "$asfDuShaf7Gafaw"
+        - in: path
+          type: string
+          name: relType
+          description: |-
+            The [relationship type](/client-server-api/#relationship-types) to search
+            for, if any. Exclude to find all child events which point to the supposed parent
+            event with any `rel_type`.
+          required: true # we can't say false, annoyingly
+          x-example: "org.example.my_relation"
+        - in: path
+          type: string
+          name: eventType
+          description: |-
+            The event type of child events to search for, if any. Exclude to find all
+            child events of any type which point to the supposed parent event.
+
+            Note that in encrypted rooms this will typically always be `m.room.encrypted`
+            regardless of the event type contained within the encrypted payload.
+          required: true # we can't say false, annoyingly
+          x-example: "m.room.message"
+        - in: query
+          type: string
+          name: from
+          description: |-
+            The pagination token to start returning results from. If not supplied, results
+            start at the earliest topological event known to the server.
+
+            Can be a `next_batch` or `prev_batch` token from a previous call, or an equivalent
+            token from [`/messages`](/client-server-api/#get_matrixclientv3roomsroomidmessages)
+            or [`/sync`](/client-server-api/#get_matrixclientv3sync) to limit results to the
+            events returned by that section of timeline.
+          required: false
+          x-example: "page2_token"
+        - in: query
+          type: string
+          name: to
+          description: |-
+            The pagination token to stop returning results at. If not supplied, results
+            continue up to `limit` or until there are no more events.
+
+            Like `from`, this can be a previous token from a prior call to this endpoint
+            or from `/messages` or `/sync` to limit to a section of timeline.
+          required: false
+          x-example: "page3_token"
+        - in: query
+          type: integer
+          name: limit
+          description: |-
+            The maximum number of results to return in a single `chunk`. The server can
+            and should apply a maximum value to this parameter to avoid large responses.
+
+            Similarly, the server should apply a default value when not supplied.
+          required: false
+          x-example: 20
+      responses:
```

Comment:
The [synapse implementation](https://github.com/matrix-org/synapse/blob/248046187940372c23466cb395b46ed97ebda1ed/synapse/handlers/relations.py#L152) does also include a field "original_event" containing the event without any aggregations. IMHO this should be included in the spec.

Reply:
Without an MSC, it can't be included here.

Reply:
> About that field, [MSC2675](https://github.com/matrix-org/matrix-spec-proposals/pull/2675) defers to [MSC2676](https://github.com/matrix-org/matrix-spec-proposals/pull/2676) where it is not mentionned (anymore?).

In that case, either this or [MSC2675](https://github.com/matrix-org/matrix-spec-proposals/pull/2675) should be changed accordingly, either adding a note for the `original_event` and it's typing, or removing it entirely.

Reply:
resolving thread: the field is destined to disappear instead. If an MSC wants to add it, a new one can be opened.